examples/elasticsearch/README.md

```diff
@@ -1,163 +1 @@
-[2015-08-30 10:01:44,042][INFO ][gateway                  ] [Hammerhead] recovered [0] indices into cluster_state
-[2015-08-30 10:08:02,517][INFO ][cluster.service          ] [Hammerhead] added {[Tenpin][2gv5MiwhRiOSsrTOF3DhuA][es-78e0s][inet[/10.244.54.4:9300]]{master=true},}, reason: zen-disco-receive(join from node[[Tenpin][2gv5MiwhRiOSsrTOF3DhuA][es-78e0s][inet[/10.244.54.4:9300]]{master=true}])
-[2015-08-30 10:10:10,645][INFO ][cluster.service          ] [Hammerhead] added {[Evilhawk][ziTq2PzYRJys43rNL2tbyg][es-rjmer][inet[/10.244.33.3:9300]]{master=true},}, reason: zen-disco-receive(join from node[[Evilhawk][ziTq2PzYRJys43rNL2tbyg][es-rjmer][inet[/10.244.33.3:9300]]{master=true}])
-```
-
-So we have a 3-node Elasticsearch cluster ready to handle more work.
-
-## Access the service
-
-*Don't forget* that services in Kubernetes are only accessible from containers in the cluster. For different behavior you should [configure the creation of an external load-balancer](http://kubernetes.io/v1.0/docs/user-guide/services.html#type-loadbalancer). While it's supported within this example service descriptor, its usage is out of scope of this document, for now.
-
-```
-$ kubectl get service elasticsearch
-NAME            LABELS                    SELECTOR                  IP(S)           PORT(S)
-elasticsearch   component=elasticsearch   component=elasticsearch   10.100.108.94   9200/TCP
-                                                                                    9300/TCP
-```
-
-From any host on your cluster (that's running `kube-proxy`), run:
-
-```
-$ curl 10.100.108.94:9200
-```
-
-You should see something similar to the following:
-
-
-```json
-{
-  "status" : 200,
-  "name" : "Hammerhead",
-  "cluster_name" : "myesdb",
-  "version" : {
-    "number" : "1.7.1",
-    "build_hash" : "b88f43fc40b0bcd7f173a1f9ee2e97816de80b19",
-    "build_timestamp" : "2015-07-29T09:54:16Z",
-    "build_snapshot" : false,
-    "lucene_version" : "4.10.4"
-  },
-  "tagline" : "You Know, for Search"
-}
-```
-
-Or if you want to check cluster information:
-
-
-```
-curl 10.100.108.94:9200/_cluster/health?pretty
-```
-
-You should see something similar to the following:
-
-```json
-{
-  "cluster_name" : "myesdb",
-  "status" : "green",
-  "timed_out" : false,
-  "number_of_nodes" : 3,
-  "number_of_data_nodes" : 3,
-  "active_primary_shards" : 0,
-  "active_shards" : 0,
-  "relocating_shards" : 0,
-  "initializing_shards" : 0,
-  "unassigned_shards" : 0,
-  "delayed_unassigned_shards" : 0,
-  "number_of_pending_tasks" : 0,
-  "number_of_in_flight_fetch" : 0
-}
-```
-
-<!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/examples/elasticsearch/README.md?pixel)]()
-<!-- END MUNGE: GENERATED_ANALYTICS -->
+This file has moved to [https://github.com/kubernetes/examples/blob/master/staging/elasticsearch/README.md](https://github.com/kubernetes/examples/blob/master/staging/elasticsearch/README.md)
```

Comment:
@timothysc we went with the same concept than when the docs/devel were moved. We leave breadcrumbs in case people linked directly to the repo. that way they know where to go:

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md